src/controls/splash.js

```diff
@@ -17,34 +16,32 @@ const Splash = function Splash(options = {}) {
     url
   } = options;
 
+  const createModal = function createModal(modalContent) {
+    return Modal({
+      title,
+      content: modalContent,
+      cls,
+      target
+    });
+  };
+
   return Component({
     name: 'splash',
     onAdd(evt) {
       viewer = evt.target;
       target = viewer.getId();
       if (!title) title = defaultTitle;
+      if (!content) content = defaultContent;
 
       if (url) {
         const fullUrl = viewer.getBaseUrl() + url;
         const req = new Request(`${fullUrl}`);
         fetch(req).then(response => response.text().then((text) => {
-          content = text;
-          this.render();
+          this.addComponent(createModal(text));
         }));
```

Comment:
Error handling would be nice. Something like:
```
.catch(e => console.warn(e)); 
```

Reply:
The linter does not like the use of console (https://eslint.org/docs/rules/no-console). I know it is in use in a few other places, but do we want to add more? Any thought on this @jokd @tonnyandersson?

Reply:
@markus659 yeah markus you are right. Maybe we should implement a error message box as a default control component and replace all console log but still have errors displayed?

Reply:
Yes, I know we talked about this previously. Origo should have some kind of error handling, errors should be displayed but maybe not by using console statements/alerts etc.

Reply:
Personally, I don't mind using console messages but perhaps we should avoid those for the sake of consistency. An error handling and displaying component sounds like a good idea, as long as it is non-intrusive, i.e. the error messages should not be displayed in a modal. I'd like to see a more subtle solution.

Reply:
I agree @tonnyandersson maybe the error component should be an icon with push message capabilities when a severe error occurs, not warning (which still could be stored but not pushed notified). The user gets notified and could click on that icon to get more info, in a menu. Any thoughts?

Reply:
Something along those lines, yeah.

Reply:
I am looking forward to your error component @daRasmussen, Origo would benefit from it.
Meanwhile maybe we can merge this and create a new issue for the error component?

Reply:
@jokd sure